README.md

```diff
@@ -35,14 +35,14 @@ https://demo.sketchbookjs.com
 3.  Run on `http://localhost:3000/`:
 
     ```
-    yarn sketchbook start
+    yarn sketchbook start example.json
```

Comment:
This is incorrect. In the previous step the file gets generated at `sketchbook/example.json`:

```suggestion
    yarn sketchbook start sketchbook/example.json
```

Reply:
Sorry this was clumsy. I've fixed it @haydn